packages/logs/README.md

```diff
@@ -148,7 +148,7 @@ Options that must have a matching configuration when using the `RUM` SDK:
 After the Datadog browser logs SDK is initialized, send a custom log entry directly to Datadog with the API:
 
 ```
-logger.debug | info | warn | error (message: string, messageContext = Context)
+logger.debug | info | warn | error (message: string, messageContext?: Context, error?: Error)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be nice to fix the signature in the onboarding snippet as well, cf https://app.datadoghq.com/logs/onboarding/client